### PR TITLE
List mintable collections on assets page

### PR DESCRIPTION
**Pull request by bobo-k2**
Display mintable collections from Polkaverse.
I merged the previous PR because I had some fixes and this new feature in different branch, so let's use this PR to resolve bugs.

Ignore files in modules folder since they are auto-generated

image

### CODE REVIEW

File: src/hooks/useCollectionInfo.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  //   { variables: { $networkName: 'Astar' } }
+  // );
+
+  const result = ref<any>([]);
+  const loading = ref<boolean>(false);
+  const error = ref<any>(null);
+
+  const URL = 'https://squid.subsquid.io/subsocial/graphql';
+
+  const fetchInfo = async (query: string) => {
```
Comment (impelcrypto) [inline src/hooks/useCollectionInfo.ts:66]:
```suggestion
  const fetchInfo = async (query: string): Promise<void> => {
```

**Review by impelcrypto — CHANGES_REQUESTED** [review]

File: src/hooks/useCollectionInfo.ts (hunk 1 above)
Comment (bobo-k2) [reply src/hooks/useCollectionInfo.ts]: Fixed here https://github.com/AstarNetwork/ink-redible-nft/pull/23/commits/c4cf1b8223750de0f86f6447099059fcf5324c42

**Review by impelcrypto — APPROVED** [review]

**Review by sirius651 — APPROVED** [review]

**Comment by Kahonnohak** [comment]
Styling request please, or please assign someone to help you.
1)  responsive behaviour on the Asset page
- Its just quite messy. I wish to achieve something very simple standard responsive card layout.
https://codepen.io/Katiae/pen/zwjmbW
I'm not going to be fussy but would be better to keep the padding the same.

2) button aligned
- Please keep the button left-aligned, and the space between buttons is 16px.
Screenshot 2023-06-12 at 16 44 54
Screenshot 2023-06-12 at 16 46 28

**Comment by Kahonnohak** [comment]
Great work @bobo-k2 

I confirm all functions works great!

Those are not very critical but good be taken care of. Whenever you have time.

- Specifics default rather than Attribute
- Token ID - Extrinsic ID
- Github - ink!redible NFT repo
- Wallet modal update (if possible or if any volunteers...)

@impelcrypto sorry to mention but do you think you could update the wallet modal...?

**Review by Kahonnohak — APPROVED** [review]
Great work!

**Pull request merged**